### PR TITLE
Bring back landscape orientation support

### PR DESCRIPTION
**Pull request by MiSikora**
## Description

This adds horizontal layouts for the player and restores support for landscape orientation.

Unfortunately, bringing this back reintroduces #3803. I will reopen the issue after this is merged. The issue is unavoidable and results from a bug in Compose. Adding the `.nestedScroll()` modifier causes incorrect velocity values to be passed to the bottom sheet callback, so we can't rely on the velocity sign to detect whether the sheet is being pulled down or not. I will report it to Google when I find time to create a sample project.

Some other minor changes:
- Increased Up Next velocity detection threshold from `1000` to `2000`. It is still a very low value, so Up Next can be easily swiped up, but a larger value helps a bit with scrolling nested content in the player.
- There are no chapter URL links in the landscape orientation with ads. Maybe we could add them, but I really don't want to spend time figuring out where to place them. It's a feature with an extremely small surface.
- The player navigation bar no longer obstructs the content. Previously, we had a solid color over the content in the player. It was a deliberate choice made during the edge-to-edge support phase, but now, with the landscape player and scrollable content, it no longer fits the UI.

> [!note]
> Do not treat the designs as pixel-perfect. They serve more as guidelines here. There are too many edge cases and platform-specific components or behaviors that the designs do not account for.

Designs: aVoIMJy9cdDZzNloBIv9jr-fi-193_17089

## Testing Instructions

Test the player in landscape orientation:
- Regular phones should display the horizontal player.
- Foldables and tablets should display the vertical player.

Obviously, your mileage may vary if a particular device falls into a different window size class.

### Pixel 6

| Audio no ads | Audio ads |
| - | - |
| n | a |
| Video no ads | Video ads |
| n | a |

### Small device

| Audio no ads | Audio ads |
| - | - |
| audio-no-ads-regular | audio-ads-regular |
| Audio no ads large font | Audio ads large font |
| audio-no-ads-large | audio-ads-large |
| Video no ads | Video ads |
| video-no-ads-regular | video-ads-regular |
| Video no ads large font | Video ads large font |
| video-no-ads-large | video-ads-large |

## Checklist
- [x] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [x] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [ ] ~I have considered whether it makes sense to add tests for my changes~
- [ ] ~All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`~
- [x] Any jetpack compose components I added or changed are covered by compose previews
- [ ] ~I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.~
 
#### I have tested any UI changes...

- [x] with different themes
- [x] with a landscape orientation
- [x] with the device set to have a large display and font size
- [x] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Class <code>NestedScrollLockableInteropConnection</code> is missing tests, but <code>unit-tests-exemption</code> label was set to ignore this.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by geekygecko — APPROVED** [review]
With so many different sizes, it's impossible with the player. Thanks for all the work you have done for this.

File: modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/nowplaying/EpisodeTitles.kt [hunk 1]
```diff
@@ -102,10 +105,11 @@ private fun Content(
     onChapterTitleClick: (Chapter) -> Unit,
     onPodcastTitleClick: () -> Unit,
     modifier: Modifier = Modifier,
+    textAlign: TextAlign = TextAlign.Center,
     playerColors: PlayerColors = MaterialTheme.theme.rememberPlayerColorsOrDefault(),
 ) {
     CompositionLocalProvider(
-        LocalRippleConfiguration provides RippleConfiguration(color = playerColors.contrast01),
+        LocalRippleConfiguration provides RippleConfiguration(Color.White, RippleDefaults.rippleAlpha(Color.White, true)),
```
Comment (MiSikora) [inline modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/nowplaying/EpisodeTitles.kt]:
```suggestion
        LocalRippleConfiguration provides RippleConfiguration(color = playerColors.contrast01),
```

File: modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/nowplaying/EpisodeTitles.kt [hunk 2]
```diff
@@ -171,8 +174,8 @@ private fun Content(
 
                         TextH70(
                             text = chapterSummary,
-                            color = playerColors.contrast02,
-                            textAlign = TextAlign.Center,
+                            color = Color.White.copy(alpha = 0.7f),
```
Comment (MiSikora) [inline modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/nowplaying/EpisodeTitles.kt]:
```suggestion
                            color = playerColors.contrast02,
```
